README.md

```diff
@@ -10,9 +10,13 @@
 
 ## Project Status
 
-As of v0.6.0, KubeOne is in the beta phase. Check out the [Backwards Compatibility Policy](./docs/backwards_compatibility_policy.md) for more details on backwards compatibility, KubeOne versioning, and maturity of each KubeOne component.
+As of v0.6.0, KubeOne is in the beta phase. Check out the [Backwards
```

Comment:
I personally don't like breaking markdown and markdown constructs just for the sake of line length, especially at like 80 characters (120 are okay nowadays). That can make some editors parse markdown incorrectly. If you have a link or something similar, make sure the whole link is on its own line.